config/a8c-for-agencies-development.json

```diff
@@ -50,7 +51,8 @@
 		"a8c-for-agencies-signup": true,
 		"a8c-for-agencies-referrals": true,
 		"a8c-for-agencies-settings": true,
-		"a8c-for-agencies-migrations": true
+		"a8c-for-agencies-migrations": true,
+		"a8c-for-agencies-automated-referrals": true
```

Comment:
I don't see this PR introducing a new `a8c-for-agencies-automated-referrals` section. Is this correct?

Reply:
Good catch! It seems a mistake.

Reply:
Good catch! 

Yes, I missed removing this!